src/main/java/org/opentripplanner/model/calendar/openinghours/OsmOpeningHoursSupport.java

```diff
@@ -43,15 +43,15 @@ private static int truncateToMinute(long startTime) {
 
   private static String toOsm(String description) {
     return switch (description.toLowerCase()) {
-      case "business days" -> "Mo-Fr";
+      case "business days" -> "Mo - Fr";
```

Comment:
https://wiki.openstreetmap.org/wiki/Key:opening_hours there isn't space around the `-`, even though it might look like there is.